docs/DART_LAB/matlab/run_lorenz_96_inf.m

```diff
@@ -165,6 +166,101 @@
 % Set the filter type string to No Assimilation to Start
 handles.filter_type_string = 'No Assimilation';
 
+
+%% Create a label for the obs Radio Group
+handles.ui_text_group_label = uicontrol('Style', 'text', ...
+    'Units'                 , 'Normalized', ...
+    'Position'              , [0.69 0.69 0.30 0.033], ...
+    'BackgroundColor'       , atts.background, ...
+    'String'                , 'Observation Network', ...
+    'HorizontalAlignment'   , 'left', ...
+    'FontName'              , atts.fontname, ...
+    'FontWeight'            , 'bold', ...
+    'FontUnits'             , 'normalized', ...
+    'FontSize'              , 0.8);
+
+%% Create a Radio Button with choices for the observation network
+
+handles.ui_button_group_obsnetwork = uibuttongroup('Visible','off', ...
+    'Position'          ,[0.63 0.60 0.35 0.08], ...
+    'BorderType', 'none', ...
+    'BackgroundColor'   , atts.background, ...
+    'FontWeight'        ,'bold', ...
+    'SelectionChangeFcn', @ObsNetwork_selection);
+
+% Create the four radio buttons in the button group.
+handles.ui_radio_full = uicontrol(handles.ui_button_group_obsnetwork, ...
+    'Style'             , 'radiobutton', ...
+    'Units'             , 'Normalized', ...
+    'Position'          , [0.12 0.6 0.4 0.4], ...
+    'BackgroundColor'   , atts.background, ...
+    'String'            , 'Full', ...
```

Comment:
I wonder if we should label these as
1:40:1   1:40:2  1:40:4   instead of   Full Half Quarter ... 

I also spent a little time (unsuccessfully) trying to figure out how to have tickmarks at the outer perimeter of the observation locations. It would be nice if you saw the observation locations - throughout the assimilation cycles, not just when you have observations.

Reply:
ACK ... When I 'reset', the observation network button was still on 1:20, but I got a full observation network when assimilating.

Reply:
ALSO ... and I don't know where ... the annotated default inflation mean is 0.6 ... below the inflation min of 1.0 ...
not sure either of these is right.